upb/def.c

```diff
@@ -364,19 +387,24 @@ bool upb_def_freeze(upb_def *const* defs, int n, upb_status *s) {
    * limit. */
   maxdepth = UPB_MAX_MESSAGE_DEPTH * 2;
 
-  /* Validation all passed; freeze the defs. */
-  ret = upb_refcounted_freeze((upb_refcounted * const *)defs, n, s, maxdepth);
```

Comment:
I changed it because they no longer are always upb_defs -- we also freeze upb_filedefs (which are not upb_defs).